src/app/dashboard/[id]/Card/Card.tsx

```
@@ -2,12 +2,24 @@ import Image from 'next/image'
 
 import type { Card } from '@/app/api/useCards'
 
+import { useDragStore } from '../store/useDragStore'
 import Tags from './Tags'
 
-export default function Card({ card }: { card: Card }) {
-  const { imageUrl, title, tags, dueDate, assignee } = card
+export default function Card({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Card'. Consider to delete it or rename it.

'Card' is defined here:


(lint/suspicious/noRedeclare)